notifications/management/commands/send_notifications.py

```diff
@@ -274,288 +308,198 @@ def find_bill_search_updates(self, search_params):
-              ON event.ocd_id = p.event_id
-            JOIN councilmatic_core_organization AS committee
-              ON p.entity_name = committee.name
-            WHERE committee.ocd_id IN %s
-              AND event.start_time > NOW()
-            ORDER BY committee.ocd_id,
-                     event.ocd_id
-            ) AS events
-            ORDER BY events.start_time DESC
-        '''
-
-        cursor = connection.cursor()
-        cursor.execute(new_events, [tuple(committee_ids)])
-        columns = [c[0] for c in cursor.description]
-
-        updates = []
-
-        grouper = lambda x: x[1]
-        event_groups = sorted(cursor, key=grouper)
-
-        for slug, group in itertools.groupby(event_groups, key=grouper):
-            group = list(group)
-
-            committee = {
-                'name': group[0][0],
-                'slug': group[0][1],
-                'events': []
-            }
-
-            for row in group:
-                event = dict(zip(columns[2:], row[2:]))
-                committee['events'].append(event)
-
-            committee['events'] = sorted(committee['events'],
-                                         key=lambda x: x['start_time'],
-                                         reverse=True)
-
-            updates.append(committee)
-
-        return updates
-
-    def find_new_events(self):
-        new_events = '''
-            SELECT * FROM (
-            SELECT DISTINCT ON (event.ocd_id)
-              event.name,
-              event.start_time,
-              event.end_time,
-              event.slug,
-              event.all_day,
-              event.location_name,
-              event.slug
-            FROM councilmatic_core_event AS event
-            JOIN new_event AS new
-              ON event.ocd_id = new.ocd_id
-            WHERE event.start_time > NOW()
-            ) AS events
-            ORDER BY events.start_time
-        '''
-
-        cursor = connection.cursor()
-        cursor.execute(new_events)
-        columns = [c[0] for c in cursor.description]
-
-        return [dict(zip(columns, r)) for r in cursor]
-
-    def find_updated_events(self):
-        new_events = '''
-            SELECT * FROM (
-            SELECT DISTINCT ON (event.ocd_id)
-              event.name,
-              event.start_time,
-              event.end_time,
-              event.slug,
-              event.all_day,
-              event.location_name,
-              event.slug
-            FROM councilmatic_core_event AS event
-            JOIN change_event AS change
-              ON event.ocd_id = change.ocd_id
-            WHERE event.start_time > NOW()
-            ) AS events
-            ORDER BY events.start_time
-        '''
+    def find_committee_event_updates(self, committee_ids, minutes=15):
+        committee_updates = []
+        for committee_id in committee_ids:
+            new_event_particip = ocd_legislative_models.EventParticipant.objects.annotate(
```

Comment:
> it's becoming clear to me that we might need to handle null end dates in all the custom managers in django-councilmatic.

It could be a good idea to standardize `django-councilmatic` model managers to provide `_dt` attributes that are cast to `DateTimeField`s with proper nulls, but I don't think I could make that change with confidence in the context of this PR. I went ahead and added/updated these attributes for models that are implicated in Notifications, however.

> could you the query councilmatic Event, whose default manager includes the datetime annotation, then grab the event participant off those objects meeting the filter criteria?

I can't believe I missed `EventManager`! This is very smart -- done.